src/resources/elements/primeDesignSystem/pinput-numeric/pinput-numeric.html

```diff
@@ -0,0 +1,28 @@
+<template
+  class="
```

Comment:
The only reason I added the classes on the template is just to make those classes show up on the component itself and not use an additional tag to do the work since it looked redundant to me. 

Eg: 
```html
<pinput-numeric class="pInput-error">
    ....
</pinput-numeric>
```
VS an additional element like:
```html
<pinput-numeric>
    <div class="pInputContainer pInput-error">
        ....
    </div>
</pinput-numeric>
```

But if that's the way you create components in this project, I will change it. Didn't know that.

Reply:
I was not aware that would work.  OK, keep it.